py_env_create

```diff
@@ -116,6 +120,8 @@ usage() {
   echo "     Run with quiet mode to minimize Conda messages. Incompatible with -v/--verbose. May fail to quiet things if you don't also specify -y/--yes."
   echo "[-r|--rename-existing] <name>  "
   echo "     If an existing environment has the same name as the environment you're installing (${new_env_name}), rename the existing env to <name>. Incompatible with -o/--overwrite."
+  echo "[-u|--update]  "
```

Comment:
Oh, I like this.

Reply:
Ugh, sorry to tease you like this, but I actually ended up removing this in a later update that I forgot to merge into this branch. The reason is that there can be subtle differences in environments introduced by updating as opposed to overwriting. I'll merge those changes in soon and request re-review.
- [ ] Do that.

Reply:
Done with 316c7fe.